### PR TITLE
seqr model fields updates

### PR DESCRIPTION
**Pull request by bw2**

### CODE REVIEW

File: xbrowse_server/base/model_utils.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    return type(obj).__module__ == "xbrowse_server.base.models"
+
+
+def find_matching_seqr_model(xbrowse_model):
+    if not _is_xbrowse_model(xbrowse_model):
+        raise ValueError("Unexpected model class: %s.%s" % (type(xbrowse_model).__module__, type(xbrowse_model).__name__))
+
+    try:
+        xbrowse_class_name = type(xbrowse_model).__name__
+
```
Comment (hanars) [inline xbrowse_server/base/model_utils.py]: Don't all the xbrowse models keep references to their seqr counterparts? For example, the xbrowse Project has a `seqr_project` field, the xbrowse Family has a `seqr_family` field, etc. Why not try those first?

File: xbrowse_server/base/views/family_views.py [hunk 2]
```diff
@@ -138,7 +139,7 @@ def edit_family(request, project_id, family_id):
             family.save()
 
             try:
-                seqr_family = get_seqr_family_from_base_family(family)
+                seqr_family = find_matching_seqr_model(family)
                 seqr_family.coded_phenotype = family.coded_phenotype
                 seqr_family.description = family.short_description
                 seqr_family.analysis_notes = family.about_family_content
```
Comment (hanars) [inline xbrowse_server/base/views/family_views.py]: I think if you used `update_xbrowse_model` above you wouldn't need to also do this here

File: xbrowse_server/gene_lists/views.py [hunk 3]
```diff
@@ -69,38 +71,39 @@ def gene_list(request, slug):
...
+                name=form.cleaned_data['name'],
+                description=form.cleaned_data['description'],
+                is_public=form.cleaned_data['is_public'],
+                last_updated = datetime.datetime.now())
+
+            GeneListItem.objects.filter(gene_list=gene_list).delete()
             for gene_id in form.cleaned_data['gene_ids']:
-                GeneListItem.objects.create(gene_list=_gene_list, gene_id=gene_id)
-            return redirect('gene_list', slug=_gene_list.slug)
+                GeneListItem.objects.create(gene_list=gene_list, gene_id=gene_id)
```
Comment (hanars) [inline xbrowse_server/gene_lists/views.py]: why isn't this using `create_xbrowse_model`?

File: xbrowse_server/base/model_utils.py [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+        'last_accessed_date': 'deprecated_last_accessed_date',
+    },
+    "Family": {
+        "family_name": "display_name",
+        "short_description": "description",
+        "about_family_content": "analysis_notes",
+        "analysis_summary_content": "analysis_summary",
+    },
+    "Individual": {
+        'gender': 'sex',
```
[the remark this answers is not in the document]
Comment (bw2) [reply xbrowse_server/base/model_utils.py]: good q : ) this was based on conversations from a while ago - sex is defined by a person's biology/genetics while gender is the person's social / self identification.

File: xbrowse_server/base/model_utils.py (hunk 1 above)
Comment (bw2) [reply xbrowse_server/base/model_utils.py]: good point. I'll change this.

Comment (bw2) [reply xbrowse_server/base/model_utils.py]: fixed.

**Comment by bw2** [comment]
@hanars if you could take another look, this should be all set

**Pull request merged**